src/components/Tokens/TokenTable/NetworkFilter.tsx

```diff
@@ -47,8 +47,8 @@ const MenuTimeFlyout = styled.span`
   max-height: 350px;
   overflow: auto;
   background-color: ${({ theme }) => theme.backgroundSurface};
-  box-shadow: ${({ theme }) => theme.flyoutDropShadow};
-  border: 1px solid ${({ theme }) => theme.backgroundOutline};
+  box-shadow: ${({ theme }) => theme.deepShadow};
+  border: 0.5px solid ${({ theme }) => theme.backgroundOutline};
```

Comment:
are we okay with .5px?

Reply:
yes that's what's the new design theme says (flyouts all have 0.5 border)